pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -329,6 +337,7 @@ var registry = func() Registry {
 	r.Register(scorePlugin1, newScorePlugin1)
 	r.Register(pluginNotImplementingScore, newPluginNotImplementingScore)
 	r.Register(duplicatePluginName, newDuplicatePlugin)
+	r.Register(testPlugin, newTestPlugin)
```

Comment:
can you just reuse the existing test plugins?

Reply:
No, old plugins only satisfied one plugin interface. In my case, I need one plugin satisfies multi-plugins and score weight doesn't be reset by other. That's why i use TestPlugin that implement all Plugin interfaces.